src/main/resources/templates/help.html

```diff
@@ -1,7 +1,9 @@
 <!DOCTYPE html>
-<html xml:th="http://www.tymeleaf.org">
+<html xmlns:th="http://www.thymeleaf.org" lang="en-US">
 <head>
-  <link rel="stylesheet" href="index.css">
+  <title>Voluntier</title>
+  <link rel="stylesheet" href="@{css/Index.css}">
```

Comment:
Moved image and CSS files being referenced to correct /resources/static directory. [Reference](https://www.baeldung.com/spring-mvc-static-resources#spring-boot)